llvm/test/CodeGen/AMDGPU/GlobalISel/sdiv.i32.ll

```diff
@@ -884,3 +786,14 @@ define <2 x i32> @v_sdiv_v2i32_24bit(<2 x i32> %num, <2 x i32> %den) {
   %result = sdiv <2 x i32> %num.mask, %den.mask
   ret <2 x i32> %result
 }
+
+define <2 x i32> @v_sdiv_i32_exact(<2 x i32> %num) {
```

Comment:
Testname is a bit misleading. It's v2i32. Do we have a scalar version of the test?